packages/browser/__tests__/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -331,6 +369,126 @@ describe("AuthCodeRedirectHandler", () => {
       ).toMatch(/^DPoP .+$/);
     });
   });
+
+  it("stores information about the resource server cookie in local storage on on successful authentication", async () => {
+    // This mocks the fetch to the Resource Server session endpoint
+    // Note: Currently, the endpoint only returns the webid in plain/text, it could
+    // be extended later to also provide the cookie expiration.
+    window.fetch = jest.fn().mockReturnValueOnce(
+      new Promise((resolve) => {
+        resolve(
+          new Response("https://my.webid", {
+            status: 200,
+          })
+        );
+      })
+    ) as jest.Mock<
+      ReturnType<typeof window.fetch>,
+      [RequestInfo, RequestInit?]
+    >;
+
+    Date.now = jest.fn().mockReturnValueOnce(10000);
+
+    const mockedStorage = mockStorageUtility({
+      "solidClientAuthenticationUser:oauth2_state_value": {
+        sessionId: "mySession",
+      },
+    });
+
+    const authCodeRedirectHandler = getAuthCodeRedirectHandler({
+      storageUtility: mockedStorage,
+    });
+
+    await authCodeRedirectHandler.handle(
+      "https://coolsite.com/?code=someCode&state=oauth2_state_value"
+    );
+    expect(
+      JSON.parse(
+        (await mockedStorage.get("tmp-resource-server-session-info")) ?? "{}"
+      )
+    ).toEqual({
+      webId: "https://my.webid",
+      sessions: {
+        "https://my.webid": {
+          expiration: 11800,
```

Comment:
This hard-coded value (which confused me at first) is based on awareness of a hard-coded DEFAULT_LIFESPAN value set elsewhere in a different module. But to keep things super-simple (since we're removing this code later), I'd just export/import that DEFAULT_LIFESPAN, and then use it here, i.e. set the 10000 value above (line 390) to a local const (e.g. `requestNowTime`), and then here have `expiration: requestNowTime + DEFAULT_LIFESPAN;`.

Reply:
60ceb8c28afaae995af2f1b5e43722ffd136cf69